test/prefer-spread.mjs

```diff
@@ -312,6 +312,8 @@ test.snapshot({
 		'buffer.slice()',
 		'file.slice()',
 		'class A {foo() {this.slice()}}',
+		'scopeManager?.scopes.slice()',
+		'(scopeManager?.scopes).slice()',
```

Comment:
I think we should report `(scopeManager?.scopes).slice()` since it's not an optional call.

Reply:
Ah it wasnt clear in the issue if the (scopeManager?.scopes).slice() was valid or invalid. Let me update the PR.